content/post/AG2021.en.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "2021 general assembly review"
+date: 2021-09-23
+draft: false
+tags: [general-assembly]
+---
+
+Like every year, we hold our general assembly. On the 4th of September, we were gathered in the Nantes area, with a few remote members, to look back upon the past year and decide on the association stances for the year to come.
```

Comment:
```suggestion
Like every year, we've held hold our general assembly. On the 4th of September, we were gathered in the Nantes area, with a few remote members, to look back upon the past year and decide on the association stances for the year to come.
```
hold for me is present tense while we need past tense. However, whether it is here or for other grammar corrections, I'm not bilingual, don't take these for 100% guaranteed.
Upcoming year, maybe? But the current formulation seems fine.

Reply:
Changed for "we held". I applied your "upcoming year" suggestion too.